operator/pkg/helmreconciler/reconciler.go

```diff
@@ -438,3 +455,19 @@ func (h *HelmReconciler) getCRNamespace() (string, error) {
 func (h *HelmReconciler) getClient() client.Client {
 	return h.client
 }
+
+func (h *HelmReconciler) addPrunedKind(gk schema.GroupKind) {
```

Comment:
is it possible to get the list of pruned object kinds from the static maps we have defined?